server/spec/queries/artists_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Artist, type: :request do
+  describe 'artists_query' do
+    let(:query_string) {
+      <<-GraphQL
+        query ($offset: Int!, $limit: Int!) {
+          artists(offset: $offset, limit: $limit) {
+            id
+            name
+          }
+        }
+      GraphQL
+    }
+    let(:result) { ServerSchema.execute(query_string, variables: { offset: 0, limit: 5 }) }
+    let(:result_data) { result['data']['artists'] }
+
+    context 'when artists exist' do
+      5.times do |n|
+        let!("artist_#{n + 1}".to_s) { create(:artist, name: "artist#{n + 1}") }
+      end
```

Comment:
`let!("artist_#{n + 1}".to_s)` これだとシンボルじゃなくて，文字列になっている。
しかも，今回は`artist_1`とかを参照しないのだから，beforeとかで作れば良いのでは？
```suggestion
      before do
        create_list(:artist, 5) do |artist, i|
          artist.name = "artist#{i+1}"
        end
      end
```

https://github.com/thoughtbot/factory_bot/issues/787#issuecomment-619814046